governance.md

```diff
@@ -44,7 +44,7 @@ SIGs and WGs follow the **lazy consensus** approach to decision-making which ass
 
 > **NOTE:** The described approach only concerns the decisions made by SIGs and WGs. It does not affect any Kyma decisions made during daily team activities.
 
-To see the current state of processed decisions in the Kyma project, go to the project's [decision board](https://app.zenhub.com/workspaces/kyma---all-repositories-5b6d5985084045741e744dea/boards?labels=decision&repos=139590666).
+To see the current state of processed decisions in the Kyma project, go to the project's [decision board](https://app.zenhub.com/workspaces/kyma---all-repositories-5b6d5985084045741e744dea/board?labels=decision&repos=139590616,139590577,139847219,139590666,139590641,147495537,151691065,167399060,169101295,171673731,150745068,167146343,165843160,170300585&showPRs=false&showClosed=false).
```

Comment:
Is there any reason that kyma/cli and kyma/compass are ommited in the zenhub link? 

```suggestion
To see the current state of processed decisions in the Kyma project, go to the project's [decision board](https://app.zenhub.com/workspaces/kyma---all-repositories-5b6d5985084045741e744dea/board?labels=decision&repos=139590616,139590577,139847219,139590666,139590641,147495537,151691065,167399060,169101295,171673731,150745068,167146343,165843160,170300585,157188288,186589820&showPRs=false&showClosed=false&showPipelineDescriptions=false).
```

Reply:
link updated